docs/migrating.md

```diff
@@ -1,296 +1,198 @@
-        on: 'user',
-        field: 'permissions',
-        permissions
-    }),
-    isPermitted,
-    (req, res, next) => {
-    // Do your thing
-    }
-);
+// feathers-authentication < v0.8.0
+const authentication = require('feathers-authentication');
+app.configure(authentication(config))
+    .use('/users', memory()) // this use to be okay to be anywhere
 ```
 
-By default this new hook and new middleware assume you are storing your permissions on a `permissions` field either as an array of strings or a string with comma separated permissions. As always, you can customize the field you are storing your permissions under so you can still use the old role based system by doing this:
+**The New Way**
 
 ```js
-const auth = require('feathers-authentication').hooks;
-userService.before({
-    all: [
-        auth.isAuthenticated(),
-        auth.checkPermissions({namespace: 'admin', on: 'user', field: 'role'})
-    ]
-});
+// feathers-authentication >= v1.0.0
+const authentication = require('feathers-authentication');
+const local = require('feathers-authentication-local');
+const jwt = require('feathers-authentication-jwt');
+const oauth1 = require('feathers-authentication-oauth1');
+const oauth2 = require('feathers-authentication-oauth2');
+
+// The services you are setting the `entity` param for need to be registered before authentication
+app.use('/users', memory())
+    .configure(authentication(config))
+    .configure(jwt(config))
+    .configure(local(config))
+    .configure(oauth1(config))
+    .configure(oauth2(config))
 ```
 
-### More Flexible Tokens (WIP)
-
-The token service has now been altered to be more generic. You can now generate different types of tokens other than JWT's. You can also generate different types of JWT's with different options separate from the standard auth token. This allows you to generate temporary access tokens for things like password reset or email confirmation or generate longer lived access tokens for things like IoT devices.
+### Config Options
 
-The service supports the following token types:
+There are a number of breaking changes since the services have been removed:
 
-- `uuid`,
-- `jwt` (default)
+- Change `auth.token` -> `auth.jwt` in your config
+- Move `auth.token.secret` -> `auth.secret`
+- `auth.token.payload` option has been removed. See [customizing JWT payload]() for how to do this.
+- `auth.idField` has been removed. It is now included in all services so we can pull it internally without you needing to specify it.
+- `auth.shouldSetupSuccessRoute` has been removed. Success redirect middleware is registered automatically but only triggers if you explicitly set a redirect. [See redirecting]() for more details.
+- `auth.shouldSetupFailureRoute` has been removed. Failure redirect middleware is registered automatically but only triggers if you explicitly set a redirect. [See redirecting]() for more details.
+- `auth.tokenEndpoint` has been removed. There isn't a token service anymore.
```

Comment:
We should add a note stating why there isn't a token service anymore.  Something like "It has been replaced by custom routes with auth strategies."